application/models/Campaign_model.php

```diff
@@ -127,7 +127,7 @@ public function prioritize_crawl($offices, $milestone)
                     $start[] = $office;
                 }
 
-                // End with offices that didn't complete last crawl
+                // End with offices that didn't complete last crawl or are long_running
```

Comment:
How is `long_running` taken into account above? I see that the value is selected, but it doesn't appear to be used here. Are you missing an `order_by` above?

Reply:
Leftover comment from initial changes. Should be removed.